cram/cram_decode.c

```diff
@@ -1689,10 +1689,13 @@ static void cram_decode_slice_xref(cram_slice *s) {
 		    int aleft = cr->apos, aright = cr->aend;
 		    int tlen;
 		    int ref = cr->ref_id;
+		    int left_cnt = 0;
```

Comment:
There's a comment in the code beneath it where it is used, but I will make it more explicit in the left_cnt variable too.